### ISSUE TITLE
Improve message on unmatched */

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#387]

**Comment by dav000** [comment]
I'm interested in working on this. I have a potential fix.

Current behavior:
```
> $ cat x.js
> var hello; */
> $ ./build/quick-lint-js x.js
x.js:1:12: error: missing operand for operator [E026]
x.js:1:13: error: unclosed regexp literal [E039]
```
Potential fix:
```
> $ ./build/quick-lint-js x.js
> x.js:1:12: error: unopened block comment [E209]
```

**Comment by strager** [comment]
Fixed by @dav000 in commit 9947942819b8798afeea93570431b48adaa42758.
